### ISSUE TITLE
[Question] Is it possible to specify a pre-existing rest-api url WITHOUT a custom domain?

### ISSUE THREAD

**Issue by Ambro17** [aws/chalice#1612]
I would like to use chalice to manage different api handlers for the same api gateway.
As far as i've seen, it isn't possible to specify a `rest_api_url`, only a custom domain. Is that right?

**Comment by jamesls** [comment]
Yes, if you're not using a custom domain, you can only set the `api_gateway_stage` which will set the prefix for your API, but you'll still use the auto-generated API gateway hostname, e.g. https://execute-api..../<your prefix>/.

**Comment by Ambro17** [comment]
I meant using an existing pre generated url

**Comment by Ambro17** [comment]
Bad bot

**Comment by jamesls** [comment]
I see, no that is not currently possible.

**Comment by Ambro17** [comment]
Would you consider adding a feature like that on the future?

**Comment by jamesls** [comment]
To clarify you're asking for support for Chalice to manage an existing REST API that you created outside of Chalice (such as in the Console, etc)?

**Comment by Ambro17** [comment]
Exactly, my particular case is one chalice app using the api gateway of another chalice app.
I find a little bit unnecessary to have one api gateway for each one and I would like to be able to share it
